app/uk/gov/hmrc/customs/managesubscription/services/SubscriptionCompleteBusinessService.scala

```diff
@@ -89,8 +89,8 @@ class SubscriptionCompleteBusinessService @Inject()(
 
   private def auditStatus(status: SubscriptionCompleteStatus, formBundleId: String)(implicit hc: HeaderCarrier): Unit =
     audit.sendDataEvent(
-      transactionName = "customs-manage-subscription-update-status",
-      path = s"/customs-manage-subscription/$formBundleId",
+      transactionName = "eori-common-component-update-status",
+      path = s"/eori-common-component/$formBundleId",
```

Comment:
Is this a "real" path?

Reply:
Yeah, there is an endpoint
```
POST        /:formBundleId                  @uk.gov.hmrc.customs.managesubscription.controllers.SubscriptionResultController.updateStatus(formBundleId)
```

Not sure why the endpoint has this first part `customs-manage-subscription` (`eori-common-component`) because in general prod routes doesn't have any prefix for endpoints, but it's possible that this is necessary from environments perspective